### PR TITLE
Forward merge master to 7.0.0

### PR DESCRIPTION
**Pull request by kusalk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -342,13 +342,6 @@
                         <skipRuntimeScope>true</skipRuntimeScope>
                     </configuration>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-assembly-plugin</artifactId>
-                    <configuration>
-                        <skipAssembly>true</skipAssembly>
-                    </configuration>
-                </plugin>
```
Comment (lukaszlenart) [inline pom.xml:351]: It's needed, in other way the final assembly packages are missing when releasing a new version of Struts 7 - no idea why it works in different way than in Struts 6

Comment (kusalk) [reply pom.xml:351]: Ah, let me revert

**Comment by kusalk** [comment]
Shit no

**Comment by kusalk** [comment]
Fixed

**Review by lukaszlenart — APPROVED** [review]

**Pull request merged**